### DISCUSSION TITLE
Sound output stops every hour or so on macOS 11.4

### DISCUSSION THREAD

**Discussion by ioanabalas** [kyleneideck/BackgroundMusic#510] [General]
My Mac just upgraded to 11.4 and Background occasionally stops working. It doesn't output any sound. I have been changing the output directly via settings to my default speakers, then back to Background and it works for another little while but then it'll stop working again. This happens around every hour or so.

**Comment by KuviX2** [comment]
For some reason the same issue has been happening to me as well.

**Comment by fabianneufang** [comment]
Same problem for me
